src/App.js

```diff
@@ -1,29 +1 @@
-import logo from './logo.svg';
 import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
```

Comment:
- [ ] There should be `a class-based App.js` component which renders the Calculator component and itself is rendered in index.js file. Kindly implement that because your current files structure is not best practice.

Reply:
Yeah! for sure @elyor-doniyorov :+1: :rocket: